department-of-veterans-affairs/caseflow
# Error message updates

Connects #6381 

### Description
This PR updates the error handling to display nicer

### Testing Plan
1. Go to /hearings/schedule/build/upload
2. Upload a spreadsheet with some errors
3. Confirm the errors look nice 😄
